packages/robo/src/cli/commands/help.ts

```diff
@@ -0,0 +1,251 @@
+	)
+	logger.log(
+		color.white(' Join our official discord at:'),
+		color.underline(color.italic(color.cyan('https://roboplay.dev/discord'))),
+		'\n'
+	)
+
+}
+
+/**
+ * Calculates the spaces for even columns
+ *
+ * @param  {number} longestCommandName - Longest command name.
+ * @param  {number} commandNameLength - Length of the Command name we are comparing it with.
+ * @returns {number} - Returns the number of spaces we have to add to the string.
+ */
+function calcSpacing(longestCommandName: number, commandNameLength: number): number {
+	let y = 0
+	if (commandNameLength === longestCommandName) {
+		return y
+	}
+	for (let i = commandNameLength; i < longestCommandName; ++i) {
+		++y
+	}
+	return y
+}
+
+// Might wanna re-work that, it splits every "70~" characters. So it's not good for every string.
+// Perhaps, adding strict grammar rules to the description might help with that.
+// lineBreakSpaces is basically the spaces I need to reach until the "-" of a command description.
+
+/**
+ * Breaks the description into smaller lines to fit the CLi and aligns them.
+ *
+ * @param {string} desc - Description of the command.
+ * @param {number} lineBreakSpaces - Number of spaces to add so the new line is aligned.
+ * @param {number} charactersToDivideInto - Number of characters the strings should be divided in.
+ * 
+ */
+function breakLine(desc: string, lineBreakSpaces: number, charactersToDivideInto: number) {
+	const numberOfDividedLines = Math.floor(desc.length / charactersToDivideInto)
+	let d = 140
+	for (let i = 0; i < numberOfDividedLines; ++i) {
+		if (i === numberOfDividedLines) {
+			logger.log('\u0020'.repeat(lineBreakSpaces), ' ' + desc.slice(d).trim())
+			logger.log('\n')
+			return
+		} else if (d === 140) {
+			logger.log('\u0020'.repeat(lineBreakSpaces), ' ' + desc.slice(68, 140).trim())
+		} else {
+			logger.log('\u0020'.repeat(lineBreakSpaces), ' ' + desc.slice(d - 70, d).trim())
+		}
+		d += 70
+	}
+}
+
+
+/**
+ * Formats the command into a structured Object. 
+ * 
+ * @param {commandGroup[]} commandGroup - Array of CommandGroup.
+ * @returns {FormattedCommand[]} - Returns the commandGroup array in the shape of FormattedCommand array.
+ */
+function formatCommand(commandGroup: CommandGroup[]): FormattedCommand[]{
+	const formattedCommands: FormattedCommand[] = [];
+
+	commandGroup.forEach((commandObject: CommandGroup, idx: number) => {
+		if(!commandObject?.command) return;
+		const childCommands = commandObject.command.getChildCommands();
+		const alias = commandObject.command
+		.getOptions()
+		.map((flags: Option) => `${flags.alias}`)
+		.join(' ')
+
+		const lastCommandInTheGroup = () => {
+			if(commandGroup[idx + 1] !== undefined && commandGroup[idx + 1].groupId !== commandObject.groupId){
+				return `${commandObject.command.getDescription()}\n\n`;
+			} else {
+				return commandObject.command.getDescription();
+			}
+		}
+
+		if(childCommands.length > 0){
+			for(let i = 0; i < childCommands.length; ++i){
```

Comment:
This could be refactored into a for-each loop. The conditional wrapping this also isn't necessary.